src/components/AppBar.js

```diff
@@ -15,7 +15,7 @@ const newTabRepo = () => {
 };
 
 const PurpleAppBar = () => (
-  <AppBar scrollHide theme={theme} rightIcon={<GithubIcon />} onRightIconClick={newTabRepo}>
+  <AppBar scrollHide flat theme={theme} rightIcon={<GithubIcon />} onRightIconClick={newTabRepo}>
```

Comment:
it comes from react-toolbox:
http://react-toolbox.com/#/components/app_bar
just disables the shadow afaik